assets/scss/main.scss

```
@@ -240,6 +240,14 @@ body {
     top: 2px;
   }
 }
+
+// Class for alignment of Counter - Intuitive Arrow Icons in the Header
```

```suggestion
// Smaller text on small mobile devices so that the header stays in one line
```

I'd change this to something that better explains the context

---

Done.